go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/databricks/databricks-sql-go v1.5.5
 	github.com/dropbox/dropbox-sdk-go-unofficial/v6 v6.0.5
 	github.com/elastic/go-elasticsearch/v8 v8.9.0
-	github.com/estuary/flow v0.3.12-0.20240118233153-7681c66d4e1f
+	github.com/estuary/flow v0.4.1-0.20240814191845-c228194b3e2f
```

Comment:
TODO: Update this pin once runtime updates are merged. This will be needed to get the tests passing.